app-benchmarks/tiobench/tiobench-0.4.2.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Portable, robust, fully-threaded I/O benchmark program"
+HOMEPAGE="https://github.com/mkuoppal/tiobench"
+SRC_URI="https://github.com/mkuoppal/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE=""
+
+RDEPEND="dev-lang/perl"
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}"/${P}-gcc5.patch )
+
+src_prepare() {
+	default
+	sed -i \
+		-e 's:/usr/local/bin:/usr/sbin:' tiobench.pl \
+		|| die "sed tiobench.pl failed"
```

Comment:
could you pls check that all perl shebangs are prefix-tractable? https://blogs.gentoo.org/mgorny/2016/02/08/a-quick-note-on-portable-shebangs/